frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/web-terminal/web-teminal-basic.ts

```diff
@@ -13,6 +14,7 @@ Then('user will see the terminal window opened in new tab', () => {
 
 And('user does nothing with displayed terminal window 1 minutes', () => {
   const terminalIdlingTimeout: number = Number(Cypress.env('TERMINAL_IDLING_TIMEOUT')) || 60000;
+  webTerminalPage.typeTextAndEnter('wtoctl set timeout 45s')
```

Comment:
Shouldn't this depend on the variable `terminalIdlingTimeout` ??

Reply:
Thx! Bind the timeouts